src/main/java/com/tcoots/CodeFellowship/config/WebSecurityConfig.java

```diff
@@ -37,31 +37,32 @@ public PasswordEncoder passwordEncoder(){
         return bCryptPasswordEncoder;
     }
 
+
     @Override
     @Bean
     public AuthenticationManager authenticationManagerBean() throws Exception {
         return super.authenticationManagerBean();
     }
 
 
+
     @Override
     protected void configure(HttpSecurity http) throws Exception {
         http
                 .csrf().disable()
                 .cors().disable()
                 .authorizeRequests()
-                    .antMatchers("/").permitAll()
                     .antMatchers(HttpMethod.POST, "/signup").permitAll()
                     .antMatchers("/*.css", "/", "/login", "/users/create", "/signup").permitAll()
                     .anyRequest().authenticated()
                 .and()
                     .formLogin()
-                .defaultSuccessUrl("/myprofile", true)
+                    .defaultSuccessUrl("/myprofile")
                     .loginPage("/login")
                     .permitAll()
                 .and()
                     .logout()
-                .logoutUrl("/")
+                    .logoutUrl("/")
```

Comment:
Delete this line. This is making your homepage log out the user every time they go to the home page.